src/Sarif.UnitTests/EqualityComparerTests.cs

```diff
@@ -33,14 +32,14 @@ public void EqualityComparer_ComputesTheSameHashCodeForDistinctEquivalentObjects
                 }
             };
 
-            Assert.AreNotSame(result1, result2);
+            Assert.NotSame(result1, result2);
```

Comment:
>Assert.NotSame [](start = 12, length = 14)

later we could consider a fluent assertions scrub, not worth it right now of course.

Reply:
I was actually going to do this but I felt this PR was big enough already.  It's a nice to have for in the future though.

---
In reply to: [141766980](https://github.com/Microsoft/sarif-sdk/pull/624#discussion_r141766980) [](ancestors = 141766980)